### DISCUSSION TITLE
Can’t connect to artisan

### DISCUSSION THREAD

**Discussion by mlinder282** [artisan-roaster-scope/artisan#795] [Q&A] [answered]
Hi, I’m looking for help, connecting my Diedrich IR2.5 to artisan. I’ve been roasting on a Hucky 500T with center 304 meter for two years with artisan 2.4, and MacBook Air. I just got my Diedrich IR2.5 with phidgets and it will not connect. Only shows 00 or uu for temps. Diedrich told me to just download lates version of artisan and phidgets. Then just pick machine:Diedrich-4 sensor.  

No good.

Please help!
Thanks 
M

**Comment by MAKOMO** [comment]
The suggest by Diedrich is correct. Just install the Phidgets driver and choose the Diedrich setup from the Config >> Machine menu in Artisan.

If you start the Phidgets Control Panel, do you see any Phidget modules in the list attaching?

If you start Artisan and connect, ensure you do not have the Phidgets Control Panel running at the same time, as only one of those two can connect to the hardware.

**Comment by mlinder282** [comment]
﻿ Hi Marko,

Thank you for offering to help. It’s truly an honor!

I deinstalled/reinstalled artisan and phidgets. Followed directions and still not working. The computer recognizes the phidgets, but still reading uu or 00.

I’m attaching photos of my screen to see if you can identify any issues?

Thank you again for your help!
Mark

[cid:3D491725-CD87-4B0F-98E9-452D45A40AEA-L0-001][cid:E48BF94E-5543-47FD-B945-D057F8009472-L0-001][cid:9E09159A-912F-400E-9AEF-8453ADCE6AC6-L0-001][cid:5004D207-CDD8-4CA0-A255-9769DBC683CC-L0-001][cid:9CCC2ECF-1D79-4D0B-A754-CD5825B93EA8-L0-001][cid:5A6F9D91-7E4F-4C86-8E72-E24E6ACA9818-L0-001][cid:6235526B-F27F-4D78-B657-A7AAD705C0F2-L0-001][cid:19B372CB-D537-477D-AE1F-52C43171EBC4-L0-001][cid:85F83585-B958-49D1-98A2-100C4B79075C-L0-001][cid:2B11C5FF-3EEF-491A-9FC7-0C7A763B6FC3-L0-001][cid:C836530F-3223-4FEC-8EDD-756F4380BD0F-L0-001][cid:F3A1CE60-BB4F-4A2E-AF6F-DFC45C0DF48B-L0-001][cid:03FB28BC-0F36-40C6-AB1D-38B87C31781A-L0-001][cid:514056FC-ABFE-4A57-BBBA-AE4EAE7EFA59-L0-001]

Sent from my iPhone

On Jan 22, 2022, at 2:13 PM, Marko Luther ***@***.***> wrote:

﻿

The suggest by Diedrich is correct. Just install the Phidgets driver and choose the Diedrich setup from the Config >> Machine menu in Artisan.

If you start the Phidgets Control Panel, do you see any Phidget modules in the list attaching?

If you start Artisan and connect, ensure you do not have the Phidgets Control Panel running at the same time, as only one of those two can connect to the hardware.

—
Reply to this email directly, view it on GitHub<https://github.com/artisan-roaster-scope/artisan/discussions/795#discussioncomment-2024428>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXNORHEOGFT3CNFJKQD62PTUXL6VRANCNFSM5MSJJO5A>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by bigl00z3** [reply]
It looks like the picture didn't load. 
Maybe try reloading it(on the GitHub discussion page, not email).

When you say that the computer is recognising the Phidgets could you confirm that:
- You see the all the Phidgets in the Phidgets control panel 
- when you double click on a Phidget (in the control panel) you see real time value changing.

Nico

**Reply by bigl00z3** [reply]
And also attach your artisan-settings here.

Export the settings "_Help/Save Settings..."_
Note that you need either add a .txt extension or zip the files before uploading. Otherwise you will receive a "Not a supported file type" error on uploading.

Nico

**Reply by mlinder282** [reply]
Good morning, sorry for the failed pictures. I will reattach. I will also send the artisan settings separately.

Thanks for all the help!
Mark
![image](https://user-images.githubusercontent.com/98232476/150683825-2cca42f2-56a5-4cbe-9dca-b168ff3916f4.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683854-05d74102-0b31-4e72-86d6-cfb141913299.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683864-e1e9f426-a214-4481-8f08-e566fc12c25e.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683878-f929e19a-22ca-4459-97cd-69dfb0e25ff0.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683883-7b94a7ab-8207-4fd8-8abd-128fda3ed10c.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683896-e495efd7-a6bb-4073-8899-08cb6efe9ae3.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683909-b6a7ea7b-b509-498c-9d39-18ca8d6ccff2.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683923-5576cbe8-6783-45e0-a344-0a1ab916a97a.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683935-8fd042b2-b164-4163-b8b6-71f03e64ee3c.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683945-fcd905bc-5b1d-43ae-8bad-d2979046e6fc.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683969-aeb5161e-0f18-4fa1-9c9d-eb5798b124b7.jpeg)
![image](https://user-images.githubusercontent.com/98232476/150683976-df39d6b6-ae32-4826-94c8-3d5137dd47f5.jpeg)![image](https://user-images.githubusercontent.com/98232476/150683995-631b22ea-e247-4c81-bc87-f24294a386b0.jpeg)![image](https://user-images.githubusercontent.com/98232476/150684003-0e124472-f9ff-4f76-b805-bac45acc8833.jpeg)![image](https://user-images.githubusercontent.com/98232476/150684007-234aa225-9899-4762-bd8d-5ef4e80b3adf.jpeg)![image](https://user-images.githubusercontent.com/98232476/150684017-064ec63a-6e90-4e9c-b787-ee0b132eed17.jpeg)

**Reply by MAKOMO** [reply]
Your settings instruct Artisan to fetch the ET/BT values from the Phidget 1048, but your Phidget Control Panel only lists the Phidget InterfaceKit 8/8/8. Thus the Phidget driver is not seeing the 1048. Most likely it is not properly connected to your USB or it is broken. I assume that Diedrich is equipping those machines with an internal USB hub to which both, the InterfaceKit module as well as the Phidget 1048 module are connected via separate USB wires. You need to fix your hardware!

**Reply by bigl00z3** [reply]
Probably Marko can explain better, but what I find surprising (I may be wrong) is that the Artisan settings are for a [Phidget 1048](https://www.phidgets.com/?tier=3&catid=14&pcid=12&prodid=1028) and in the Phidget control panel it is a [Phidget InterfaceKit 8/8/8](https://www.phidgets.com/?prodid=1021). 

Could that be linked the problem?

In the Phidget control panel if you double click on the Voltage Input lines do you see any signal?

I doubt that would fix, but have you tried the 3 different Diedrich settings in Artisan?
<img width="473" alt="Screenshot 2022-01-23 at 20 11 51" src="https://user-images.githubusercontent.com/8886922/150694283-c6625814-1278-4b3c-8065-0e43119708d5.png">

And also, as previously mentioned above by Marko when you test in Artisan, always make sure that the Phidget Control panel is closed. Otherwise it can't work.

Nico

**Comment by mlinder282** [comment]
Hi Nico,
Here are my artisan settings. Also since I could not connect the Diedrich, I used my Hucky, switch to Center 304 meter, that worked correctly. 

Thanks,
Mark
[artisan-settings.aset.zip](https://github.com/artisan-roaster-scope/artisan/files/7921117/artisan-settings.aset.zip)

**Comment by mlinder282** [comment]
Hi Marko and Nico,

Thank you for helping me evaluate the problem. 
I have tried the following three Diedrich settings. 
4 sensor- gives no readings
6 sensor-Power shows 72.6
6 sensor pre 2018-Gas usage shows 278.7

I have also verified that the Phidget control is closed prior to opening Artisan.

When double clicking on the Voltage input, I get the following readings:
VOLTAGE INPUT 0 = 3.5638 (this value dropped to .5861 when the blower fan and gas was turned on)
VOLTAGE INPUT 1 = 0
VOLTAGE INPUT 2 = 0
VOLTAGE INPUT 3 =5.0 (does not change value)
VOLTAGE INPUT 4 =0
VOLTAGE INPUT 5 = 0
VOLTAGE INPUT 6 = 0
VOLTAGE INPUT 7 = 0

Diedrich supplied the following USB cable: Tripp-Lite, Model U024-006, USB 2.0 Gold Extension Cable A/A(Male/Female) 6ft.

Again many thanks for your time!

![Screen Shot 2022-01-23 at 2 42 59 PM](https://user-images.githubusercontent.com/98232476/150696206-1213315a-b3b9-4ab9-8da8-7c2f6b7f31c8.png)
![Screen Shot 2022-01-23 at 2 42 15 PM](https://user-images.githubusercontent.com/98232476/150696207-50378cff-45e4-45c9-bed1-a3b98e7b7bb3.png)
![Screen Shot 2022-01-23 at 2 41 21 PM](https://user-images.githubusercontent.com/98232476/150696208-aba01c09-21fe-4734-955d-dad2d601e8c0.png)

**Reply by bigl00z3** [reply]
First try with an other usb cable. 

If it doesn't work, try to find the location of the Phidgets in the roaster and connect directly to the Phidget 1048 to see if the computer sees it you can leave the roaster off while you test like this as the usb will power the Phidget).
The goal of the tests is to see if it's the Phidget itself that is the issue or if it's something along the line (usb hub, cable...)

Nico

**Reply by mlinder282** [reply]
Thank you for all the help! I've ordered a new cable and will test again tomorrow.

Best Regards,
Mark

**Comment by mlinder282** [comment, marked as answer]
HI Marko & Nico,

I received the new usb cable today. At 1st it was the same result. only showing Phidget 8/8/8, so I opened the control panel, and re-seated all usb connections.  At that point is showed Phidget 8/8/8 and Phidget 4 sensor!  It seems all is operational with the exception of board temp.  I think we are almost there!

Thank you for all the help!
Mark
![Screen Shot 2022-01-24 at 7 07 19 PM](https://user-images.githubusercontent.com/98232476/150888043-88e6b6b6-85e0-4b32-a8f8-b1fd73570cbe.png)
![Screen Shot 2022-01-24 at 7 07 48 PM](https://user-images.githubusercontent.com/98232476/150888046-14db018e-d88d-49f5-9e17-53378cbfa6dd.png)
![Screen Shot 2022-01-24 at 7 07 59 PM](https://user-images.githubusercontent.com/98232476/150888047-c12392b7-9454-4b4c-99fc-37eb27303fbf.png)
![Screen Shot 2022-01-24 at 7 13 45 PM](https://user-images.githubusercontent.com/98232476/150888049-f886b0de-e44b-4045-ac29-96929e62d9e1.png)

**Reply by mlinder282** [reply]
Can you advise if this looks good? Also what is the function of board temp? 
Thanks again for all the support!
Mark

**Reply by MAKOMO** [reply]
Board Temp is taken from the Phidget 1048 internal temperature sensor and indicates the temperature of the electronic. This is about ambient temperature, but might rise over time. It is used as internal temperature reference for the TC temperature calculation. For you this is totally unimportant. Still I wonder why this is not reported as obviously the TC temperatures are all reported correctly. Strange.

Further, as your machine has a Phidget InterfaceKit installed, you might actually have a 6-Sensor data logging hardware, so you should use the corresponding setup (not sure if pre-2018 or the newer setting works correctly for you.
